index.html

```diff
@@ -227,6 +136,43 @@ <h3 class="deskt">Let's Chat!</h3>
         </div>           
     </section>
 
+    <!-- The Modal -->
+    <div id="myModal" class="modal">
```

Comment:
- [ ] Nice working on your modal popup, kindly add a background just like you have on the Figma Template
- [ ] Also kindly dynamically populate the information of the modal with the data in the project array.